package.json

```diff
@@ -0,0 +1,66 @@
+{
+    "name": "invoice-express-api",
+    "version": "0.1.0",
+    "description": "The RIPE Invoice Express API client",
+    "keywords": [
+        "api",
+        "client",
+        "invoice-express",
+        "js"
+    ],
+    "homepage": "https://github.com/ripe-tech/invoice-express-api-js#readme",
+    "bugs": {
+        "url": "https://github.com/ripe-tech/invoice-express-api-js/issues"
+    },
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/ripe-tech/invoice-express-api-js.git"
+    },
+    "license": "Apache-2.0",
+    "author": {
+        "name": "Platforme",
+        "url": "https://www.platforme.com"
+    },
+    "main": "dist/invoice-express-api.cjs.js",
+    "unpkg": "dist/invoice-express-api.umd.js",
+    "module": "dist/invoice-express-api.esm.js",
+    "browser": "dist/invoice-express-api.umd.js",
+    "files": [
+        "LICENSE",
+        "dist/**/*",
+        "res/**/*",
+        "js/**/*.js",
+        "test/**/*.js"
+    ],
+    "scripts": {
+        "build": "rollup -c",
+        "example": "node examples/simple.js",
+        "lint": "eslint rollup.config.js \"./{js,test}/**/*.{js,json}\"",
+        "lint-fix": "eslint rollup.config.js \"./{js,test}/**/*.{js,json}\" --fix",
+        "prettier": "prettier rollup.config.js \"./*.{js,json}\" \"./{js,test}/**/*.{js,json}\" --write",
+        "pretty": "npm run prettier && npm run lint-fix",
+        "test": "mocha --recursive",
+        "upgrade": "npx sort-package-json && ncu -u",
+        "watch": "rollup -c -w"
+    },
+    "dependencies": {
+        "yonius": "^0.6.5"

```

Comment:
If possible, let's start with upgraded dependencies maybe. At least for yonius (maybe others as well)